Makefile

```diff
@@ -181,6 +181,7 @@ export KUBEBUILDER_ASSETS=$(PROJECT_DIR)/bin
 
 .PHONY: test
 test: generate-mocks lint bin/ginkgo bin/kubectl bin/kube-apiserver bin/etcd ## Run tests. At the moment this is only unit tests.
+	@./hack/testing_ginkgo_recover_statements.sh --add # Add ginkgo.GinkgoRecover() statements to controllers.
```

Comment:
Reverting. This has to run before the test suite or the test suite will have already loaded the package without the ginkgo statements.